internal/ui/backup/text_test.go

```diff
@@ -0,0 +1,47 @@
+package backup
+
+import (
+	"testing"
+
+	"github.com/restic/restic/internal/errors"
+	"github.com/restic/restic/internal/test"
+)
+
+type mockTerm struct {
```

Comment:
That `mockTerm` looks identical to the one used in `ui/restore`. Instead of duplicating them, please move it to `internal/ui/mock.go` (we could also create a subfolder, but that feels like overkill).